src/components/common/fullCalendar/processEvents.tsx

```diff
@@ -8,18 +8,33 @@ interface EventData {
 	classNames: string;
 }
 
-const processEvents = (timeBlockData: TimeBlockData): EventData[] => {
+interface TasksEventData {
+	taskId: number;
+	timeBlockId: number;
+	title: string;
+	start: string;
+	end: string;
+	allDay?: boolean;
+	classNames: string;
+}
```

Comment:
```suggestion
interface TasksEventData extends EventData {
	taskId: number;
	timeBlockId: number;
}
```

여기 인터페이스 상속으로 이렇게 써줄 수 있을 것 같네요!

Reply:
오 오 이렇게도 할 수 있군요 배우고 갑니닷 👍👍👍